package.json

```diff
@@ -13,38 +13,38 @@
         "deploy-dev": "npm run deploy -- deploy.dev.yml"
     },
     "dependencies": {
-        "@navikt/ds-css": "^0.16.4",
-        "@navikt/ds-react": "^0.17.4",
-        "@navikt/nav-dekoratoren-moduler": "^1.6.16",
-        "amplitude-js": "^8.17.0",
-        "classnames": "^2.3.1",
-        "jsonwebtoken": "^8.5.1",
-        "jwks-rsa": "^2.0.5",
-        "lodash.debounce": "^4.0.8",
-        "next": "^12.1.6",
-        "node-cache": "^5.1.2",
-        "node-fetch": "^2.6.7",
-        "node-schedule": "^2.1.0",
-        "react": "^17.0.2",
-        "react-dom": "^17.0.2"
+        "@navikt/ds-css": "1.3.5",
+        "@navikt/ds-react": "1.3.5",
+        "@navikt/nav-dekoratoren-moduler": "1.6.9",
```

Comment:
Det finnes nyere versjoner av denne i Github pakke-repo'et. Mulig du må logge inn lokalt for å se den, `npm login npm.pkg.github.com` eller noe sånt :)

Reply:
Ja, den hadde ikke VS Code plugin fått med seg. Skal være fikset i commit 7772729.